server/odc-core/src/main/java/com/oceanbase/odc/core/sql/execute/mapper/MySQLTimestampMapper.java

```diff
@@ -41,10 +41,11 @@ public Object mapCell(@NonNull CellData data) throws SQLException {
         if (!isValidTimestamp(data)) {
             return "0000-00-00 00:00:00";
         }
-        String returnValue = timestamp.toString();
-        if (returnValue.endsWith(".0")) {
-            return returnValue.substring(0, returnValue.length() - 2);
-        }
+
+        String returnValue = new String(data.getBytes());
```

Comment:
remove useless code, you may not put them in a comment~

and you can return it directly.